pkg/stanza/operator/input/file/testdata/logs-1-lines.log

```diff
@@ -0,0 +1 @@
+Vx8jhicmw58iT8dOZcp0V4sNA+fEX1nzmicjxyqmZHPcrBiG7txZUfV3OsMyRlef
```

Comment:
Do we need to commit these files? Elsewhere we generate them before starting the benchmark.

Reply:
Changed in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/38054/commits/5d4b3f0c079bc011c96b71724923730ba7ae3187.